package.json

```diff
@@ -2,7 +2,7 @@
   "name": "tex2max",
   "version": "1.3.1",
   "description": "LaTeX math to Maxima code converter",
-  "main": "lib/tex2max.common.js",
+  "main": "build/merge.js",
```

Comment:
was this change intended?

Reply:
Yes, it's needed for correct `tex2max` import and `lib` directory does not even exist in the repo and is probably an output directory of the `build` script.

Reply:
So I managed to build tex2max and it works with `lib/tex2max.common.js` as well. Should I add the lib folder to git?

Reply:
what do you mean it works with `lib/tex2max.common.js` is this a dependency? if it is inlcuded via a dependency you don't need to add it otherwise what would happen if we don't add the lib folder? is building even possible without it?

Reply:
No, lib not a dependency. It is a directory created by building tex2max. Similar to mathquill, ckeditor5 and probably all the custom node modules, we track the build directories with version control. `lib/tex2max.common.js` is the built tex2max library that should be used by the webapp.

Reply:
so does not work without the lib folder and the build is not part of the deploy?

Reply:
I see, I guess than this totally fine